.circleci/config.yml

```diff
@@ -1,15 +1,15 @@
 version: 2.1
 
-plain-go114: &plain-go116
```

Comment:
Do you know why this said 1.14 previously?

Reply:
Because that was the minimum version we tested with before. It seems we forgot to change the name when we accepted the version support policy that deprecated 1.14